closed/autoconf/custom-hook.m4

```diff
@@ -46,14 +46,15 @@ AC_DEFUN_ONCE([CUSTOM_EARLY_HOOK],
   OPENJ9_CONFIGURE_CUDA
   OPENJ9_CONFIGURE_DDR
   OPENJ9_CONFIGURE_DEMOS
-  OPENJ9_CONFIGURE_HEALTHCENTER
-  OPENJ9_CONFIGURE_NUMA
-  OPENJ9_CONFIGURE_WARNINGS
+  OPENJ9_CONFIGURE_JFR
   OPENJ9_CONFIGURE_JITSERVER
+  OPENJ9_CONFIGURE_HEALTHCENTER
   OPENJ9_CONFIGURE_INLINE_TYPES
-  OPENJ9_THIRD_PARTY_REQUIREMENTS
   OPENJ9_CHECK_NASM_VERSION
+  OPENJ9_CONFIGURE_NUMA
+  OPENJ9_THIRD_PARTY_REQUIREMENTS
   OPENJCEPLUS_SETUP
+  OPENJ9_CONFIGURE_WARNINGS
```

Comment:
Any particular reason for moving `OPENJ9_CONFIGURE_WARNINGS` to the end? it is not in alphabetical order.